### ISSUE TITLE
Why ONE-PEACE as the abbreviation?

### ISSUE THREAD

**Issue by cmhungsteve** [OFA-Sys/ONE-PEACE#2]
I'm curious why this work is called ONE-PEACE.
Is it an abbreviation of something?
If it is about the anime One Piece, then why not call it ONE-PIECE?

Thanks.

**Comment by logicwong** [comment]
Firstly, I am a fan of One Piece. This idea came to me while reading the manga, so I gave it a similar name.
Secondly, the core of One Piece is freedom. The initial goal of this work was to create the most "free" model that could be used in infinite modalities. In this regard, they are similar.
Lastly, I believe that as the ultimate treasure of Laugh Tale, "One Piece" is equal to "Want Peace", it hopes that those who eventually reach Laugh Tale will stand up against the tyranny of Heavenly Celestial Dragon and bring the world towards peace.
Therefore, we changed "Piece" to "Peace" and named it One-Peace.
